services/backend/src/commands/System/about.ts

```diff
@@ -43,18 +51,46 @@ export class StatsCommand extends Command {
       (line) => `${Emojis.bulletPoint} ${line}`
     );
 
+    const commandStats =
+      await this.container.client.prisma.commandStats.findMany();
+
+    const formatted = commandStats
+      .sort((a, b) => b.uses - a.uses)
+      .map((c, i) => `${i + 1}. ${c.name}: ${c.uses}`)
+      .filter((_, i) => i < 5);
```

Comment:
This is really messy, [ninja code](https://javascript.info/ninja-code) isn't really the style we're going for.

Reply:
It would also be nice to make a util to parse numbers so 1000 would become 1,000.

Reply:
Can't we simply use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

Reply:
> Can't we simply use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

Yep, sounds good.